packages/sanity/src/core/form/inputs/PortableText/PortableTextInput.tsx

```diff
@@ -343,7 +338,7 @@ export function PortableTextInput(props: PortableTextInputProps) {
               onChange={handleEditorChange}
               maxBlocks={undefined} // TODO: from schema?
               readOnly={readOnly}
-              type={type as FIXME}
+              type={type}
```

Comment:
We should make a note for the future to rename this prop to `schemaType` for consistency

Reply:
Yes, I did at least keep it as `schemaType` inside that component (was renamed from props).